docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -0,0 +1,92 @@
+---
+
+This guide describes the general workflow for making documentation changes
+
+## Step 1/4 Set up your local environment
+Clone the `gravitational/next` repository via `git clone`:
+
+```code
+$ git clone https://github.com/gravitational/next
+```
+
+Different versions of the documentation are organized into the `content` directory of `gravitational/next`. Each subdirectory of `content` includes a git submodule for a different branch of `gravitational/teleport`. 
+
+Navigate to the root of your local clone of the `gravitational/next` repository and run the following command to populate the `content` directory:
+
+```code
+$ git submodule update --init --recursive
+```
+
+Next, navigate to the directory under `content` that corresponds to the latest version of Teleport.
+
+```code
+$ cd content/(=version=)
+```
+
+Check out a new branch of `gravitational/teleport`.
+
+```code
+$ git checkout -b my-branch
+```
+
+## Step 2/4 Test your changes
+
+Run the following command to start the development server:
+
+```code
+$ yarn dev
+```
+
+You can visit the docs site at `localhost:3000/docs`. The development server will detect changes to the documentation automatically as you make them and refresh the page within the browser.
+   
+Validate your changes locally using the provided linters:
+
+```code
+# Check your Markdown syntax
+$ yarn markdown-lint
+# markdown-lint, but also checks that external links work
+$ yarn markdown-lint-external-links
+```
+
+## Step 3/4 Create a pull request
+Commit your changes and push your branch to `gravitational/teleport` (if you work at Teleport) or a fork of `gravitational/teleport`.
+
+```code
+# If you are an external contributor, create a fork and push to it.
+# This command assumes that you have created an SSH key pair and stored 
+# your public key in GitHub.
+$ git remote add fork ssh://git@github.com/my-user/teleport
+$ git rebase origin/master
+$ git push -u fork my-branch
+```
+
+Visit https://github.com/gravitational/teleport and create a pull request (PR) against your branch. Our continuous integration (CI) pipeline will automatically assign reviewers to your PR.
+
+## Step 4/4 Backport your changes
+Since our docs site is versioned, we want to ensure that any changes you make to the most recent version of our documentation are reflected for the versions of Teleport we currently support.
+
+<Admonition type="tip" title="Supported Versions">
+You can find our list of currently supported versions in the FAQ:
+https://goteleport.com/docs/faq/#which-version-of-teleport-is-supported
+</Admonition>
+
+We will illustrate a backport for Teleport version 8.x.x. In this example, we have created a PR against `origin/master` from the branch `my-branch`, which has been committed to our fork.
+
+Copy the hash of the commit we made to `my-branch`, then run the following commands:
+
+```code
+$ git fetch origin
+$ git checkout branch/v8
```

Comment:
Should we recommend the more modern git commands for this? `git switch` exists because `git checkout` is confusing to new users.

```
$ git switch branch/v8
$ git switch -c my-branch-backport-v8
$ git cherry-pick ...

```